plugins/catalog-react/src/components/EntityTable/presets.tsx

```diff
@@ -28,7 +27,6 @@ import {
 
 export const systemEntityColumns: TableColumn<SystemEntity>[] = [
   createEntityRefColumn({ defaultKind: 'system' }),
-  createDomainColumn(),
```

Comment:
I don't thin that it's the right way to remove it from the presets. We should create a custom column layout in the table